### PR TITLE
V15 fix links

### LINKED ISSUE #1171: Broken internal links - v15
Part of the https://github.com/vitessio/website/issues/1169 umbrella to fix broken links

Below is a truncated output of broken internal links -- filtered for v15 results.

```
➜  vitess.io git:(prod) make check-internal-links
rm -rf public
npm install

up to date, audited 176 packages in 941ms

40 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npx hugo --cleanDestinationDir -e dev -DFE
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguage
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguageinsubdir
WARN 2022/10/12 10:24:31 Unknown Sitemap field: disablelanguages
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguage
WARN 2022/10/12 10:24:31 Unknown Sitemap field: defaultcontentlanguageinsubdir
WARN 2022/10/12 10:24:31 Unknown Sitemap field: disablelanguages
Start building sites …
hugo v0.96.0-2fd4a7d3d6845e75f8b8ae3a2a7bd91438967bbb+extended darwin/arm64 BuildDate=2022-03-26T09:15:58Z VendorInfo=gohugoio

                   |  EN  | ZH
-------------------+------+------
  Pages            | 1664 | 396
  Paginator pages  |    0 |   0
  Non-page files   |  108 |  12
  Static files     |  120 | 120
  Processed images |    0 |   0
  Aliases          |  734 |   0
  Sitemaps         |    2 |   1
  Cleaned          |    0 |   0

Total in 2502 ms
curl https://raw.githubusercontent.com/wjdp/htmltest/master/godownloader.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10278  100 10278    0     0  57657      0 --:--:-- --:--:-- --:--:-- 59755
wjdp/htmltest info checking GitHub for latest tag
wjdp/htmltest info found version: 0.16.0 for v0.16.0/macos/arm64
wjdp/htmltest info installed ./bin/htmltest
bin/htmltest
htmltest started at 10:24:37 on public/docs
========================================================================

⋮

15.0/user-guides/vschema-guide/backfill-vindexes/index.html
  hash does not exist --- 15.0/user-guides/vschema-guide/backfill-vindexes/index.html --> #CreateLookupVindex
15.0/user-guides/vschema-guide/subsharding-vindex/index.html
  target does not exist --- 15.0/user-guides/vschema-guide/subsharding-vindex/index.html --> ../../../reference/compatibility/query_planner
15.0/user-guides/schema-changes/managed-online-schema-changes/index.html
  target does not exist --- 15.0/user-guides/schema-changes/managed-online-schema-changes/index.html --> ../../../design-docs/table-lifecycle/safe-lazy-drop-tables/
  target does not exist --- 15.0/user-guides/schema-changes/managed-online-schema-changes/index.html --> ../../../design-docs/online-ddl/scheduler
15.0/user-guides/schema-changes/recoverable-migrations/index.html
  target does not exist --- 15.0/user-guides/schema-changes/recoverable-migrations/index.html --> ../../../design-docs/online-ddl/scheduler
15.0/user-guides/sql/vtexplain-in-bulk/index.html
  target does not exist --- 15.0/user-guides/sql/vtexplain-in-bulk/index.html --> ../../../contributing
15.0/user-guides/sql/vtexplain/index.html
  target does not exist --- 15.0/user-guides/sql/vtexplain/index.html --> ./explain-format-vtexplain
  target does not exist --- 15.0/user-guides/sql/vtexplain/index.html --> ../../../contributing
15.0/reference/vreplication/metrics/index.html
  hash does not exist --- 15.0/reference/vreplication/metrics/index.html --> ../../features/monitoring/#3-push-based-metrics-system#3-push-based-metrics-system
15.0/reference/vreplication/flags/index.html
  target does not exist --- 15.0/reference/vreplication/flags/index.html --> ../../../design-docs/vreplication/vstream/tracker/
15.0/reference/vreplication/progress/index.html
  target does not exist --- 15.0/reference/vreplication/progress/index.html --> ../../../design-docs/vreplication/life-of-a-stream
15.0/reference/vreplication/movetables/index.html
  target does not exist --- 15.0/reference/vreplication/movetables/index.html --> ../../../../../docs/user-guides/configuration-advanced/unmanaged-tablet/#move-legacytable-to-the-commerce-keyspace
  target does not exist --- 15.0/reference/vreplication/movetables/index.html --> ../../../../../docs/user-guides/migration/move-tables/
  target does not exist --- 15.0/reference/vreplication/movetables/index.html --> ../../../../../docs/concepts/move-tables/
15.0/reference/features/schema-management/index.html
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#getschema
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#listalltablets
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#validateschemashard
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#validateschemakeyspace
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#getvschema
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#getsrvvschema
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#applyvschema
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#applyvschema
  hash does not exist --- 15.0/reference/features/schema-management/index.html --> ../../../reference/programs/vtctl/#rebuildvschemagraph
15.0/reference/features/mysql-replication/index.html
  hash does not exist --- 15.0/reference/features/mysql-replication/index.html --> ../../programs/vtctl/#emergencyreparentshard
15.0/reference/features/vindexes/index.html
  target does not exist --- 15.0/reference/features/vindexes/index.html --> ../../../design-docs/query-serving/clookup-vindex/
15.0/reference/programs/vtctl/index.html
  hash does not exist --- 15.0/reference/programs/vtctl/index.html --> ../vtctl/keyspaces#removekeyspacesell
  hash does not exist --- 15.0/reference/programs/vtctl/index.html --> ../vtctl/keyspaces#createLookupvindex
  hash does not exist --- 15.0/reference/programs/vtctl/index.html --> ../vtctl/keyspaces#VDiff
  hash does not exist --- 15.0/reference/programs/vtctl/index.html --> ../vtctl/serving-graph#getsrvsvchema
15.0/reference/vtctldclient-transition/overview/index.html
  hash does not exist --- 15.0/reference/vtctldclient-transition/overview/index.html --> ../../programs/vtctld/#grpc-vtctld-mdash-new-in-v14
15.0/get-started/index.html
  target does not exist --- 15.0/get-started/index.html --> ../contributing/build-on-centos
  target does not exist --- 15.0/get-started/index.html --> ../contributing/build-on-macos
  target does not exist --- 15.0/get-started/index.html --> ../contributing/build-on-ubuntu
15.0/get-started/local-brew/index.html
  target does not exist --- 15.0/get-started/local-brew/index.html --> ../../../overview/supported-databases/
15.0/get-started/local-mac/index.html
  target does not exist --- 15.0/get-started/local-mac/index.html --> ../../../overview/supported-databases/
========================================================================
✘✘✘ failed in 1.056837833s
270 errors in 1571 documents
make: *** [run-link-checker] Error 1
```

### PR DESCRIPTION
**Pull request by dogukanteber**
Partly fixes #1171. Some links are temporarily skipped due to the absence of `VStream API` page in v15. Below is the result of `make check-internal-links` command:

```
dogukan@dogukan:~/website$ make check-internal-links | grep 15.0
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10278  100 10278    0     0   5868      0  0:00:01  0:00:01 --:--:--  5866
wjdp/htmltest info checking GitHub for latest tag
wjdp/htmltest info found version: 0.17.0 for v0.17.0/linux/amd64
wjdp/htmltest info installed ./bin/htmltest
15.0/reference/vreplication/internal/vstream-stream-migration/index.html
  target does not exist --- 15.0/reference/vreplication/internal/vstream-stream-migration/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/vstream-stream-migration/index.html --> ../../vstream/
15.0/reference/vreplication/internal/life-of-a-stream/index.html
  target does not exist --- 15.0/reference/vreplication/internal/life-of-a-stream/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/life-of-a-stream/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/life-of-a-stream/index.html --> ../../vstream/
15.0/reference/vreplication/internal/vstream-skew-detection/index.html
  target does not exist --- 15.0/reference/vreplication/internal/vstream-skew-detection/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/vstream-skew-detection/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/vstream-skew-detection/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/vstream-skew-detection/index.html --> ../../vstream/#minimizeskew
  target does not exist --- 15.0/reference/vreplication/internal/vstream-skew-detection/index.html --> ../../vstream/
  target does not exist --- 15.0/reference/vreplication/internal/vstream-skew-detection/index.html --> ../../vstream/
253 errors in 1580 documents
make: *** [Makefile:37: run-link-checker] Error 1
```
This PR will be updated when it is decided how to replace `VStream API` file.

### CODE REVIEW

**Comment by mattlord** [comment]
Thanks, @dogukanteber ! ❤️

File: content/en/docs/15.0/reference/features/vindexes.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The lookup table that implements a Lookup Vindex can be sharded or unsharded.  N
...
 
-Consistent lookup vindexes use an alternate approach that makes use of careful locking and transaction sequences to guarantee consistency without using 2PC. This gives the best of both worlds, with the benefit of a consistent cross-shard vindex without paying the price of 2PC. To read more about what makes a consistent lookup vindex different from a standard lookup vindex read our [consistent lookup vindexes design documentation](../../../../design-docs/query-serving/clookup-vindex).
+Consistent lookup vindexes use an alternate approach that makes use of careful locking and transaction sequences to guarantee consistency without using 2PC. This gives the best of both worlds, with the benefit of a consistent cross-shard vindex without paying the price of 2PC. To read more about what makes a consistent lookup vindex different from a standard lookup vindex read our [consistent lookup vindexes design documentation](../../../../design-docs/query-serving).
```
Comment (mattlord) [inline content/en/docs/15.0/reference/features/vindexes.md]: This should go directly to the RFC in GitHub now: https://github.com/vitessio/vitess/issues/4855

File: content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ As general overview:
 - Tablets will independently run schema migrations:
   - `ALTER TABLE` statements run via `VReplication`, `gh-ost` or `pt-online-schema-change`, as per selected [strategy](../ddl-strategies)
   - `CREATE TABLE` statements run directly.
-  - `DROP TABLE` statements run [safely and lazily](../../../design-docs/table-lifecycle/safe-lazy-drop-tables/).
+  - `DROP TABLE` statements run [safely and lazily](https://github.com/vitessio/vitess/blob/main/doc/design-docs/SafeLazyDropTables.md).
```
Comment (mattlord) [inline content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md:31]: I'm not sure if this is right now or not. @deepthi, do you know?

File: content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ A migration can be in any one of these states:
 
 A migration is said to be _pending_ if we expect it to run and complete. Pending migrations are those in `queued`, `ready` and `running` states.
 
-For more about internals of the scheduler and how migration states are controlled, see [Online DDL Scheduler](../../../design-docs/online-ddl/scheduler)
+For more about internals of the scheduler and how migration states are controlled, see [Online DDL Scheduler](https://github.com/vitessio/vitess/blob/main/doc/design-docs/OnlineDDLScheduler.md)
```
Comment (mattlord) [inline content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md:165]: I'm also not sure if this is right or not. @deepthi or @shlomi-noach ? Note that this link is corrected in quite a few pages here.

**Review by mattlord — COMMENTED** [review]
There were a few links that I'm not sure about, but everything else looks good (I double checked the links on each page in the netlify preview). Thank you again!

File: content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md (hunk 2 above)
Comment (mattlord) [reply content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md:31]: I confirmed that it's what we're now using here: https://vitess.io/docs/design-docs/online-ddl/

So we're good!

File: content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md (hunk 3 above)
Comment (mattlord) [reply content/en/docs/15.0/user-guides/schema-changes/managed-online-schema-changes.md:165]: I confirmed that it's what we're now using here: https://vitess.io/docs/design-docs/online-ddl/

So we're good!

**Review by mattlord — APPROVED** [review]

File: content/en/docs/15.0/reference/features/vindexes.md (hunk 1 above)
Comment (mattlord) [reply content/en/docs/15.0/reference/features/vindexes.md]: This is the only thing to fix now.

**Review by mattlord — CHANGES_REQUESTED** [review]

File: content/en/docs/15.0/reference/features/vindexes.md (hunk 1 above)
Comment (frouioui) [reply content/en/docs/15.0/reference/features/vindexes.md:67]: This will be fixed by https://github.com/vitessio/website/pull/1361

Comment (frouioui) [reply content/en/docs/15.0/reference/features/vindexes.md]: I agree with what @mattlord said. We shall link to https://github.com/vitessio/vitess/issues/4855 directly.

**Review by frouioui — COMMENTED** [review]
Please update `content/en/docs/15.0/reference/features/vindexes.md` in this PR

**Comment by dogukanteber** [comment]
I made the necessary changes. Just to let you guys know

**Review by mattlord — APPROVED** [review]
Thanks again, @dogukanteber ! ❤️

**Comment by mattlord** [comment]
@dogukanteber would you mind doing an empty commit+push? The GitHub action is off in the weeds and I can't do anything in these cases (happens sometimes) other than push, but I can't push to your fork. 🙂 Sorry.

**Pull request merged**